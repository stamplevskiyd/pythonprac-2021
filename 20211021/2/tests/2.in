:output x print([x % i for i in range(1, 11)])
:count y y * 3 + y[::-1] * 2
count 'abc'
output 100
quit "That's all"

