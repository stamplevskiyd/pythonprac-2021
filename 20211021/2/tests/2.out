abcabcabccbacba
[0, 0, 1, 0, 0, 4, 2, 4, 1, 0]
None
3
